backend/src/database/migrations/20221012054742-remove-api-key-column-from-user-table.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, _) => {
+    await queryInterface.removeColumn('users', 'api_key')
+    await queryInterface.sequelize.query(
+      'DROP TRIGGER format_user_trigger ON users;'
+    )
+    await queryInterface.dropFunction('format_user_on_insert', [])
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.createFunction(
+      'format_user_on_insert', // actually a misnomer, since activates on UPDATE too, but stick to existing naming
+      [],
+      'trigger',
+      'plpgsql',
+      `
+      NEW.email = LOWER(NEW.email);
+      NEW.api_key = CASE WHEN NEW.api_key='' THEN NULL ELSE NEW.api_key END;
```

Comment:
Incorrect trigger function body

Reply:
Should set `api_key_hash = api_key`

Reply:
ohh I think in this case your [previous comment](https://github.com/opengovsg/postmangovsg/pull/1841/files#r994058139) is correct, in the down migration, I should create the column and backfill first, before creating the trigger and function

(let me test again locally to make sure)

Reply:
alternatively, lmk (via Slack) if we should hop on a call to discuss this (none of this is particularly urgent tho, so no hurry)